### ISSUE TITLE
Allow to exclude models from forestadmin

### ISSUE THREAD

**Issue by storm23** [ForestAdmin/laravel-forestadmin#42]
## Expected behavior

I need to exclude models from forestadmin. 

How to do this ?
1) With a property in Eloquent models :
```
//Models/MyModel.php
<?php

protected $ignoreForestAdmin = true; //default is false
```
2) In config file
```
//config/forest.php
<?php
return [
   'ignore_models' => [
         'Models\MultyTenancy' //namespace to ignore
   ]
];
```

## Actual behavior

In my use case, some models are linked to an "on the fly" database (multi tenancy architecture). When I launch the server I have an error for models linked to this database (which not exists in this context).

## Failure Logs

 ```
InvalidArgumentException 

  Database connection [client] not configured.

  at vendor/laravel/framework/src/Illuminate/Database/DatabaseManager.php:161
    157▕         // If the configuration doesn't exist, we'll throw an exception and bail.
    158▕         $connections = $this->app['config']['database.connections'];
    159▕ 
    160▕         if (is_null($config = Arr::get($connections, $name))) {
  ➜ 161▕             throw new InvalidArgumentException("Database connection [{$name}] not configured.");
    162▕         }
    163▕ 
    164▕         return (new ConfigurationUrlParser)
    165▕                     ->parseConfiguration($config);

      +13 vendor frames 
  14  artisan:37
      Illuminate\Foundation\Console\Kernel::handle()
```

**Comment by matthv** [comment]
Thanks for the feedback @storm23.
This is a feature that will be implemented very soon.

**Comment by matthv** [comment]
The feature is released in the 1.0.0-beta.21 version.

More information : https://docs.forestadmin.com/documentation/how-tos/settings/include-exclude-models
